codingGuideLines/cpp.md

```diff
@@ -590,3 +590,12 @@ template<
 >
 using GetValueType_t = typename traits::GetValueType< T_Type >::type;
 ```
+
+
+## 20. Special CUDA Syntax
+  - it is not allowed to add spaces within the CUDA introduced brackets `<<<` and `>>>`
```

Comment:
use same form as above + add highlighting:
  - you **must not** add spaces *between* the individual brackets of the CUDA kernel brackets `<<<` and `>>>`